homework-6/script-1.js

```diff
@@ -0,0 +1,83 @@
+//Написати функцію, яка приймає 4 аргументи і шукає найменше число серед них
+const getMinNumber = (num_1, num_2, num_3, num_4) => {
+    if (!isNaN(num_1) && !isNaN(num_2) && !isNaN(num_3) && !isNaN(num_4) && num_1 !== null && num_2 !== null && num_3 !== null && num_4 !== null && num_1 !== undefined && num_2 !== undefined && num_3 !== undefined && num_4 !== undefined) {
```

Comment:
All this can be simplified 
```suggestion
    if (Number.isInteger(num_1) && Number.isInteger(num_2) && Number.isInteger(num_3) && Number.isInteger(num_4)) {
```

<img width="566" alt="image" src="https://user-images.githubusercontent.com/84010501/191562085-5010c318-a70e-476a-8b7e-7f430b32f978.png">